src/tabcenter.js

```diff
@@ -50,12 +50,24 @@ TabCenter.prototype = {
         browser.tabs.create({});
       }
     });
-    this._newTabButtonView.addEventListener("mousedown", () => {
-      this._longPressTimer = setTimeout(() => {
-        if (browser.contextualIdentities) {
+    this._newTabButtonView.addEventListener("mousedown", async e => {
+      switch (e.which) {
+      case 1: {
+        this._longPressTimer = setTimeout(() => {
           this.showNewTabMenu();
-        }
-      }, LONG_PRESS_DELAY);
+        }, LONG_PRESS_DELAY);
+        break;
+      }
+      case 2: {
+        let currentTab = await browser.tabs.query({ active: true })[0];
```

Comment:
Feel free to merge these 2 patches together and close 1 PR, it will make reviewing way easier